### ISSUE TITLE
Bug Report: Current and Needed Ants

### ISSUE THREAD

**Issue by ghost** [PlaceholderGames/2016-VnD_Game#60]
The HUD isn’t displaying the how many active ants there are on level and the number of ants needed to reach the finish line.

This bug can be fixed by creating a variable on each level to store the number of ants needed to finish a level and every second counting how many ant actors are active on a level, doing it every frame would slow the game down.

**Comment by ghost** [comment]
This bug has been fixed, the HUD is now displaying the current number of ant actors on the level in real time and ants needed to complete the level which is stored on each level as the “ants_needed” variable.

**Comment by ghost** [comment]
This issue is ready for review!

**Comment by napiorek** [comment]
Tested, seems to work fine

![image](https://cloud.githubusercontent.com/assets/22525895/23623204/0d91f5be-0299-11e7-9bf4-12b1f5674da6.png)

**Comment by BojanStankovic** [comment]
Tested this and merged manually since the modifications used level's internal macros. I recreated the modifications using the external macro libraries. @Richard-Chaos we have been using Macro libraries that live outside of level blueprint. In case you are not sure how it works just let me know and I'll explain it to you. It's quite straightforward 😃 

NOTE: the maximum amount of ants shown on the HUD must be set separately from the ant spawner that actually determines that amount.

Closing
